### PR TITLE
feat: add one base method to the EntityContainer extension

### PR DESCRIPTION
**Pull request by nlunets**
- We now support getEntityInterface on the base API for the EntityContainer.js
- Slightly improved the documentation to be more helpful hopefully :)

### CODE REVIEW

File: docs/DefiningMockdata.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
+# Defining mockdata
+
+
+## Influencing entity set behavior
+
+The mockserver allows you to define your mock data as javascript file and function that allow you to influence the behavior of the standard function to match your needs.
+
+In order to define your mockdata, you need to create a file with the name of the entity set you want to mock and the extension `.js` in the folder identifier by your `mockdataPath`.
```
Comment (drktfl) [inline docs/DefiningMockdata.md:8]: ...in the folder identifie**D**

File: docs/DefiningMockdata.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+
+You can then implement any of the function defined in the [API](./MockserverAPI.md) to add your custom logic.
```
Comment (drktfl) [inline docs/DefiningMockdata.md:10]: > any of the function

"some of the functions"? I think "any" means "all of them"

File: docs/EntityContainerAPI.md [hunk 3]
```diff
@@ -0,0 +1,38 @@
+# EntityContainer contributor API
+
+When defining your own unbound actions, you can provide a javascript (or Typescript) file with the name `EntityContainer` that will implement various actions.
```
Comment (drktfl) [inline docs/EntityContainerAPI.md:3]: EntityContainer.js

**Review by drktfl — DISMISSED** [review]
definitely an improvement :)

File: docs/DefiningMockdata.md (hunk 1 above)
Comment (nlunets) [reply docs/DefiningMockdata.md:8]: ack

File: docs/DefiningMockdata.md (hunk 2 above)
Comment (nlunets) [reply docs/DefiningMockdata.md:10]: `some functions` seems to be a good alternative

File: docs/EntityContainerAPI.md (hunk 3 above)
Comment (nlunets) [reply docs/EntityContainerAPI.md:3]: ack

**Review by drktfl — APPROVED** [review]

**Pull request merged**